src/components/Alteration/AlterationSummary.vue

```diff
@@ -143,8 +143,6 @@
             </p>
 
             <EffectiveDateTime
-              :currentJsDate="getCurrentJsDate"
-              :effectiveDateTime="getEffectiveDateTime"
```

Comment:
EffectiveDateTime already used store getters so I just added a couple more getters and removed these props.